cli/slashingExport.go

```diff
@@ -36,13 +36,35 @@ func SlashingExportCmd(sedgeActions actions.SedgeActions) *cobra.Command {
 	)
 
 	cmd := &cobra.Command{
-		Use:   "slashing-export [flags]",
+		Use:   "slashing-export [validator]",
 		Short: "Export slashing protection data",
-		Long:  "Export slashing protection interchange data (EIP-3076). Validator is stopped if is currently running",
+		Long: `
+Export Slashing Protection Interchange Format (EIP-3076) data. This command assumes
```

Comment:
Nice help. Explain the [validator] operand and add an example

Reply:
Done https://github.com/NethermindEth/sedge/pull/209/commits/21c6518944592287704424f7cbc34e8ee26ef921